packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/extensions/DevMenuExtension.kt

```diff
@@ -137,18 +138,51 @@ class DevMenuExtension(reactContext: ReactApplicationContext)
   override fun devMenuScreens(): List<DevMenuScreen>? {
     return listOf(
       screen("testScreen") {
-        action("reload", {}) {
-          label = { "Reload" }
-          glyphName = { "reload" }
-          keyCommand = KeyCommand(KeyEvent.KEYCODE_R)
-          importance = DevMenuItemImportance.HIGHEST.value
-        }
-
-        action("reload", {}) {
-          label = { "Reload" }
-          glyphName = { "reload" }
-          keyCommand = KeyCommand(KeyEvent.KEYCODE_R)
-          importance = DevMenuItemImportance.HIGHEST.value
+        group {
+          selectionList {
+            addItem(DevMenuSelectionList.Item().apply {
+              isChecked = { true }
+              title = { "release-1.0" }
+              warning = { "You are currently running an older development client version than the latest \"release-1.0\" update. To get the latest, upgrade this development client app." }
+              tags = {
+                listOf(
+                  DevMenuSelectionList.Item.Tag().apply {
+                    glyphName = { "ios-git-network" }
+                    text = { "production" }
+                  },
+                  DevMenuSelectionList.Item.Tag().apply {
+                    glyphName = { "ios-cloud" }
+                    text = { "90%" }
```

Comment:
What does this value mean? Is it a progress of downloading the bundle? If so, I guess it doesn't make much sense as the items are static and we don't update them when the bottom sheet is already open 🤔